healthcare/healthcare/doctype/diagnostic_report/diagnostic_report.py

```
@@ -5,7 +5,38 @@
+def get_diagnostic_report_title(patient_name, age, gender, observation_category=None):
+	title = get_diagnostic_report_base_title(patient_name, age, gender)
+	if observation_category:
+		title += CATEGORY_TITLE_MARKER.format(observation_category)
+	return title
+
+
+def get_observation_category_from_title(title):
+	if not title or "[Observation Category:" not in title:
+		return None
+
+	prefix, _, suffix = title.rpartition("[Observation Category:")
```

Ruff (0.15.13)

[warning] Unpacked variable `prefix` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)